### ISSUE TITLE
Allow System Title DLC and Updates

### ISSUE THREAD

**Issue by gudenau** [Cruel/freeShop#173]
It would be handy to allow system titles to have updates and DLC downloaded, like the themes and street pass Mii plaza.

**Comment by cheatfreak47** [comment]
That's already implemented.

**Comment by gudenau** [comment]
I didn't see that anywhere in the UI, where do I find that?

**Comment by cheatfreak47** [comment]
in the list of installed titles

**Comment by gudenau** [comment]
Interesting, I could not find the street pass update or dlc in there.

**Comment by Ziul** [comment]
There is a way to split the DLC into more blocks? Actually,Home menu themes are splited into 2 blocks and it's almost impossible to set all then. Install only the second block is possible, butt the first ( with 274 themes) always give me fails on installing. Would be nice if they are splited into blocks of 30~40.

**Comment by br1980** [comment]
Is that comment about needing to reinstall DLC the same for updates as well?  I don't see updates for system titles or carts.

I also don't see DLC for carts either (same issue potentially?)

Finally, the theme list only allows 100 themes to be installed at the same time, so if you download the first set you're prompted to delete themes one by one (or all at once by deleting the entire thing in FreeShop).

**Comment by Cruel** [comment]
If you have an update for a game already installed, and there is another update released for it, you will need to reinstall the update. And for carts, in case you didn't know, you need to put the cart in the console so it shows up in the game list. If it doesn't show, it's because freeShop doesn't have that particular game (a problem for cart-only released games).

**Comment by br1980** [comment]
Thanks for the info.  If I have purchased DLC for a game but I want to get the rest, from your description it sounds like I need to delete the dlc then install again?

**Comment by Cruel** [comment]
Yeah, @br1980 it shows it as installed even if it's only partial. So if you're missing any, need to reinstall it.
